src/main.ts

```diff
@@ -38,6 +41,28 @@ Vue.use(Vuetify)
 Vue.use(Affix)
 Vue.use(Vuelidate)
 Vue.use(Vue2Filters)
+// Default options - https://github.com/apostrophecms/sanitize-html (under Default options)
+Vue.use(VSanitize)
+const vuetify = new Vuetify({
+  iconfont: 'mdi',
```

Comment:
Was this moved out for some specific reason? Trying to understand if why themes are loaded first/before start()

Reply:
because we needed to include Vue.use for TiptapVuetify and also the new Vue instance

Reply:
Okay. thanks